sdk/purview/azure-purview-administration/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0b1 (2021-09-26)
+
+- This is the initial release of the Azure Purview Administration library.
```

Comment:
I think it might be worth it to mention that Azure Purview Account is included in this library

Reply:
Yes, we maye want to write something  "This package also includes the operations that were previously released part of azure-purview-account"